mywebsite/app_news/urls.py

```diff
@@ -9,4 +8,6 @@
     path('', ArticleListView.as_view(), name='article_list'),
     path('<int:year>/<int:month>/<int:day>/<slug:article_slg>',
          article_detail, name='article_detail'),
```

Comment:
Ок, но читабельнике будет так
```suggestion
    path(
        '<int:year>/<int:month>/<int:day>/<slug:article_slg>',
        article_detail,
        name='article_detail',
    ),
```

Reply:
исправлено cf515df15e7b92619ec6ed492b1fddd1c98b005b